markets/perps-market/contracts/utils/OffchainUtil.sol

```diff
@@ -44,11 +44,11 @@ library OffchainUtil {
         updateData[0] = result;
 
         IPythVerifier verifier = IPythVerifier(settlementStrategy.priceVerificationContract);
-        uint256 msgValue = verifier.getUpdateFee(1);
+        uint256 msgValue = verifier.getUpdateFee(updateData);
```

Comment:
```suggestion
        uint256 msgValue = msg.value
```
To be clear, I think this would work just fine

Reply:
Mmmm... for this, it can be tricky since we can receive founds (msg.value) not only to be sent to Pyth verifier, if we do that we are diverging all the funds to this call.